packages/ppxx/ppxx.1.2.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+version: "1.2.1"
+authors: "Jun Furuse"
+maintainer: "jun.furuse@gmail.com"
+bug-reports: "jun.furuse@gmail.com"
```

Comment:
If you could put a `mailto:` prefix on this, that would be nice for surfacing in web UIs.